.circleci/config.yml

```diff
@@ -337,6 +346,60 @@ jobs:
             - "dist/*linux_amd64.deb"
 
 
+  test-e2e-webui:
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: large
+    steps:
+      - checkout
+      - run: sudo apt-get update && sudo apt-get install -y docker-ce docker-ce-cli containerd.io
```

Comment:
we should use https://github.com/determined-ai/determined/blob/master/.circleci/config.yml#L54 for this line and the one below